API/ev3_lcd.c

```diff
@@ -1847,3 +1847,38 @@ int LcdPrintf(char color, const char *fmt, ...)
 
 	return c - buf;
 }
+
+int Ev3Printf(const char *fmt, ...)
+{
+	va_list args;
+	char *buffer;
+
+	va_start(args, fmt);
+	vasprintf(&buffer, fmt, args);
+	va_end(args);
+
+	int ret = LcdPrintf(1, buffer);	// LcdPrintf with color=1 for black text

```

Comment:
I would prefer if you added a LcdvPrintf which both `LcdPrintf` and `Ev3Printf` can use. This would avoid having the format string parsed twice.

You also don't `free` the memory allocated by vasprintf here. (I just noted LcdPrintf doesn't either...)